Lib/idlelib/idle_test/test_calltip.py

```diff
@@ -98,8 +98,8 @@ def test_signature_wrap(self):
             self.assertEqual(get_spec(textwrap.TextWrapper), '''\
 (width=70, initial_indent='', subsequent_indent='', expand_tabs=True,
     replace_whitespace=True, fix_sentence_endings=False, break_long_words=True,
-    drop_whitespace=True, break_on_hyphens=True, tabsize=8, *, max_lines=None,
-    placeholder=' [...]')
+    drop_whitespace=True, break_on_hyphens=True, tabsize=8, fold_space_newline=False,
+    *, max_lines=None, placeholder=' [...]')
```

Comment:
If change is needed to pass, so be it.  First PR on issue, #5615 did not happen to require change here.  A new option likely means that this would not be backported.  If so, please add this comment.
```suggestion
    *, max_lines=None, placeholder=' [...]')  # Test revised for 3.11, bpo-32397.
```
I cannot otherwise review. Decision between competing PRs should be made on the issue. I added people involved with #5615 as potential reviews of this.

Reply:
Yep, this was needed to pass the tests.. I've commented on the Issue in regard to other PR vs. this one; thanks for taking a look and commit suggestion, looks good :)